frontend/simple/views/Invite.vue

```diff
@@ -164,21 +272,72 @@ export default {
             },
             mailbox
           )
-          await backend.publishLogEntry(member.state.attributes.mailbox, invite)
-
-          // We need to make a record of the invitation in the group's contract
-          const latest = await backend.latestHash(this.$store.state.currentGroupId)
-          const invited = new Events.HashableGroupRecordInvitation(
-            {
-              username: member.state.attributes.name,
-              inviteHash: invite.toHash(),
-              sentDate
-            },
-            latest
-          )
-          await backend.publishLogEntry(this.$store.state.currentGroupId, invited)
+          // We need to determine if the invitation needs a vote
+          const groupLatest = await backend.latestHash(this.$store.state.currentGroupId)
```

Comment:
When there are two logical sections (i.e. a logical branch), as there is on the next line, try to keep all of the resources/activities/function-calls that are unique to one logical branch confined within that logical branch.

I.e. here `groupLatest` is only used within `this.contract.members.length >= 3` context, and so it is better to move it into that context.

This helps:

- The reviewer/maintainer/code-reader to better understand the code / what is used where / what is needed for what / etc.
- It has the added benefit of making the code slightly more performant by eliminating unnecessary steps in the case where the branch is not taken